src/zepben/eas/client/feeder_load_analysis_input.py

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from dataclasses import dataclass
+from typing import List, Optional
+
+__all__ = [
+    "FeederLoadAnalysisInput"
+]
+
+
+@dataclass
+class FeederLoadAnalysisInput:
+    """ A data class representing the configuration for a feeder load analysis study """
+
+    feeders: Optional[List[str]]
+    """The mRIDs of feeders to solve for feeder load analysis"""
+
+    substations: Optional[List[str]]
+    """The mRIDs of substations to solve for feeder load analysis"""
+
+    subGeographicalRegions: Optional[List[str]]
+    """The mRIDs of sub-Geographical Region to solve for feeder load analysis"""
+
+    geographicalRegions: Optional[List[str]]
+    """The mRIDs of Geographical Region to solve for feeder load analysis"""
```

Comment:
These aren't being passed through to the GQL query.

Reply:
added and fixed naming convention to be python standard. Will change up example.